ipynb/primary-processing/8-collapse_samples.md

```diff
@@ -0,0 +1,210 @@
+Some of the per-results figures require various slices and perspectives of the data. This notebook performs these necessary additional summarizations.
+
+```python
+>>> import os
+...
+>>> import americangut.notebook_environment as agenv
+>>> import americangut.util as agu
+...
+>>> import qiime_default_reference as qdr
+...
+>>> chp_path = agenv.activate('8')
+```
+
+Let's make sure we have the paths we need.
+
+```python
+>>> ag_100nt_biom = agu.get_existing_path(agenv.paths['ag-100nt-biom'])
+>>> ag_cleaned_md = agu.get_existing_path(agenv.paths['ag-cleaned-md'])
+```
+
+And let's setup all the paths that we're going to create.
+
+```python
+>>> ag_100nt_1k_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-biom'])
+>>> ag_100nt_1k_fecal_biom = agu.get_new_path(agenv.paths['ag-100nt-1k-fecal-biom'])
+>>> ag_100nt_1k_skin_biom  = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-biom'])
+>>> ag_100nt_1k_oral_biom  = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-biom'])
+...
+>>> ag_100nt_1k_fecal_sex_biom  = agu.get_new_path(agenv.paths['ag-100nt-1k-fecal-sex-biom'])
+>>> ag_100nt_1k_fecal_diet_biom = agu.get_new_path(agenv.paths['ag-100nt-1k-fecal-diet-biom'])
+>>> ag_100nt_1k_fecal_age_biom  = agu.get_new_path(agenv.paths['ag-100nt-1k-fecal-age-biom'])
+>>> ag_100nt_1k_fecal_bmi_biom  = agu.get_new_path(agenv.paths['ag-100nt-1k-fecal-bmi-biom'])
+...
+>>> ag_100nt_1k_oral_sex_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-sex-biom'])
+>>> ag_100nt_1k_oral_diet_biom      = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-diet-biom'])
+>>> ag_100nt_1k_oral_age_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-age-biom'])
+>>> ag_100nt_1k_oral_flossing_biom  = agu.get_new_path(agenv.paths['ag-100nt-1k-oral-flossing-biom'])
+...
+>>> ag_100nt_1k_skin_sex_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-sex-biom'])
+>>> ag_100nt_1k_skin_cosmetics_biom = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-cosmetics-biom'])
+>>> ag_100nt_1k_skin_age_biom       = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-age-biom'])
+>>> ag_100nt_1k_skin_hand_biom      = agu.get_new_path(agenv.paths['ag-100nt-1k-skin-hand-biom'])
+...
+>>> ignored = 'foo'
```

Comment:
Can add a comment if you'd like, but there was a file path that was necessary for the call but not used after the fact